test/test_sparse.py

```diff
@@ -1999,6 +2000,31 @@ def _test_sparse_mask_shape(self, nnz_x1, nnz_x2, shape_i, shape_v, dtype, devic
         self.assertEqual(self.safeToDense(y1), expected)
         self.assertEqual(self.safeToDense(y2), expected)
 
+    @dtypes(torch.double, torch.cdouble)
+    @skipIfCrossRef
+    def test_sparse_mask_backward(self, device, dtype):
+        from itertools import product, repeat
+
+        shape = (5, 5)
+        sparse_dims = len(shape)
+        nnzs = (0, 5, 15, 25)
+
+        lhs_data = torch.arange(25, device=device).reshape(shape).to(dtype).to_sparse(sparse_dims)
```

Comment:
Notice that `lhs_data._nnz() == 24`. Suggest using:
```suggestion
        lhs_data = torch.arange(1, 26, device=device).reshape(shape).to(dtype).to_sparse(sparse_dims)
```

Reply:
Not sure I understand. I want to have these zeroes materialized.